# type
org.test.demo.type.internal.error.server=error interno del servidor
org.test.demo.type.parse.json=JSON mal formado
org.test.demo.type.resource.not.found=recurso no encontrado
org.test.demo.type.resource.duplicate.key=clave duplicada
org.test.demo.type.resource.constraint.violation=violación de restricción
org.test.demo.type.illegal.argument=argumento no válido
org.test.demo.type.resource.data.integrity.violation=integridad referencial
org.test.demo.type.method.not.supported=método no soportado

# message default
org.test.demo.message.no.resource.for.path=no hay recurso para la url {0}
org.test.demo.message.resource.constraint.violation=el contenido enviado contiene {0} error(es) de validación

# message errorInfo
org.test.demo.message.errorInfo.code.null=el código del error no puede ser null
org.test.demo.message.errorInfo.not.found=no se ha encontrado el error con código {0}

# message category
org.test.demo.message.category.id.null=el id de la categoría no puede ser null
org.test.demo.message.category.not.found=no se ha encontrado la categoría con id {0}
org.test.demo.message.category.duplicate.key=ya existe una categoría con el nombre {0}
org.test.demo.message.category.data.integrity=existen subcategorías que referencian a la categoría con id {0}

# message subcategory
org.test.demo.message.subcategory.id.null=el id de la subcategoría no puede ser null
org.test.demo.message.subcategory.not.found=no se ha encontrado la subcategoría con id {0}
org.test.demo.message.subcategory.duplicate.key=ya existe una subcategoría con el nombre {0} y la categoría id {1}


